compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/native/ComposeLayer.jsNative.kt

```diff
@@ -68,10 +70,13 @@ internal class ComposeLayer(
                 SkikoTouchEventKind.MOVED,
                 SkikoTouchEventKind.CANCELLED,
                 SkikoTouchEventKind.ENDED -> {
+                    val scale = density.density
                     scene.sendPointerEvent(
                         eventType = event.kind.toCompose(),
-                        // TODO: account for the proper density.
-                        position = Offset(event.x.toFloat(), event.y.toFloat()) - getTopLeftOffset(), // * scene.density
+                        position = Offset(
+                            x = event.x.toFloat() * scale,
+                            y = event.y.toFloat() * scale
+                        ) - getTopLeftOffset(),
```

Comment:
Does `getTopLeftOffset` return value in pixels or in points? We should check if screen shifting works correctly, when the virtual keyboard appears.

Reply:
getTopLeftOffset() returns value in points.
I added a fix to the keyboard in this PR: https://github.com/JetBrains/androidx/pull/361

Reply:
@igordmn Maybe it is better to change a returned type?
```getTopLeftOffset(): Offset``` -> ```getTopLeftOffset(): DpOffset```

Reply:
Good idea, let's do that. `Offset` is usually associated with pixels.

Reply:
Ok, I will change it in separate branch!

Reply:
Made simpler. Leaved ```getTopLeftOffset(): Offset```, but convert values inside UIKit implementation to pixels.
(https://github.com/JetBrains/androidx/pull/361#discussion_r1060908536)

Reply:
Dima's PR has been merged into this PR